libsrc/supervision/crt0.s

```diff
@@ -67,7 +67,10 @@ not_dma:
 ; Removing this segment gives only a warning.
         .segment "FFF0"
 .proc reset32kcode
-        lda     #(6<<5)
+        lda     #$A0
+        sta     lcd_width
+        sta     lcd_height
+        lda     #$C9
```

Comment:
Why `%11000000` instead of the `(6<<5)` of the original code?

Reply:
Why (6<<5) does not work as verified on Wataroo, Mame and REAL hardware by Plamen @PlamenVaysilov. The fix has been equally verified on real real hardware by Plamen.

Remark: Potator 0.6/0.7 does not show the problem.

Reply:
Okay, trying to be more explicit:

* The original code is `#(6<<5)`

* You suggest `#$C9`

* @polluks suggests `#%11000000 | SV_LCD_ON | SV_NMI_ENABLE_ON` instead

* I ask why not `#(6<<5) | SV_LCD_ON | SV_NMI_ENABLE_ON`

Reply:
Sorry, #$C9 was suggested by me by @PlamenVaysilov who is a real expert and who has a multi-cart for the Supervision and has tested this on real hardware positively as well tested the bug.
I can try #%11000000 but I would like to have it tested on real hardware if Plamen is available. Let me see...

Reply:
Okay, trying to be _EVEN_ more explicit:

`#(6<<5)` equals `#$C0`. You have benn told that `#$C9` is better than `#$C0`. Nobody disagrees so far.

But instead of writing `#$C9` you can as well write `#%11000000 | SV_LCD_ON | SV_NMI_ENABLE_ON`. It's the same value, it's just easier to understand (no "magic constant").

And instead of `#%11000000 | SV_LCD_ON | SV_NMI_ENABLE_ON` you can as well write `#(6<<5) | SV_LCD_ON | SV_NMI_ENABLE_ON`. It's still the same value, but maybe even more easy to understand.

Reply:
I agree. Sorry for not understanding you... I am a bit tired.